debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -508,8 +508,13 @@ protected Object convertPoint(Column column, Field fieldDefn, Object data){
         if (data == null) {
             data = fieldDefn.schema().defaultValue();
         }
-
+        
```

Comment:
I am actually wondering about my configuration for my IDE, I am using Intellij IDEA. I will try to to take care of these trailing whitespace in my next PR :)